kratos/modeler/modeler.h

```diff
@@ -68,85 +48,47 @@ class Modeler
     typedef std::size_t SizeType;
     typedef std::size_t IndexType;
 
-    //typedef ModelPart::GeometricalDataContainerType GeometricalDataContainerType;
-
-    //typedef GeometricalDataContainerType::GeometricalDataType GeometricalDataType;
-
-    //typedef ModelPart::GeometryType GeometryType;
-
-    //typedef ModelPart::GeometriesContainerType GeometriesContainerType;
-
-
     ///@}
     ///@name Life Cycle
     ///@{
 
     /// Default constructor.
-    Modeler() {}
+    Modeler(
+        Model& rModel,
+        const Parameters ModelerParameters = Parameters())
+        : mParameters(ModelerParameters)
+    {}
 
     /// Destructor.
-    virtual ~Modeler() {}
-
+    virtual ~Modeler() = default;
 
     ///@}
-    ///@name Operators
+    ///@name Modeler Stages at Initialize
     ///@{
 
+    /// Import geometry models from external input.
+    virtual void ImportGeometryModel() const
```

Comment:
These methods cannot be const

Reply:
sure. thanks for the comment. It is changed